README.md

```diff
@@ -58,6 +58,10 @@ Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
+## Aknowledgements
+
+Hats off to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) for providing the design for this project!
+
```

Comment:
- [ ] Great job but I can not find a presentation video of your project. Please record a maximum 5 mins video presenting the features of the project you built as required for this project +1
![196996460-fe0a77f2-d190-40f3-86a9-d1945102cc73](https://user-images.githubusercontent.com/6615394/197361749-51cbae5c-4eb0-4be6-8c21-0c5fedd34896.png)